crates/tauri-cli/src/build.rs

```diff
@@ -172,19 +172,36 @@ pub fn setup(
         ));
     }
 
+    // Issue #13287 - Allow the use of target dir inside frontendDist/distDir
+    // https://github.com/tauri-apps/tauri/issues/13287
+    let mut target_path = get_cargo_target_dir(&options.args)?;
+    target_path = tauri_path.join(target_path);
+    let target_path = fs::canonicalize(target_path)?;
```

Comment:
This section doesn't make much sense. get_cargo_target_dir will return an absolute path and so will overwrite tauri_path. joining it to the tauri_path may not be correct for relative paths either i think.